<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite guice-stage="DEVELOPMENT" name="Failed suite [Default suite]">
  <test thread-count="5" name="Default test(failed)">
    <classes>
      <class name="com.revature.testng.SalesforceLoginTest">
        <methods>
          <include name="findLoginPageSuccessful"/>
          <include name="afterTest"/>
          <include name="beforeSuite"/>
          <include name="afterMethod"/>
          <include name="loginAsVpSuccessful"/>
          <include name="beforeClass"/>
          <include name="loginAsTrainerLandsOnHomePage"/>
          <include name="loginAsTrainerWithWrongCredentials"/>
          <include name="beforeMethod"/>
          <include name="loginAsTrainerSuccessful"/>
          <include name="afterClass"/>
          <include name="afterSuite"/>
          <include name="beforeTest"/>
        </methods>
      </class> <!-- com.revature.testng.SalesforceLoginTest -->
    </classes>
  </test> <!-- Default test(failed) -->
</suite> <!-- Failed suite [Default suite] -->
